src/ctia/stores/es/crud.clj

```diff
@@ -464,14 +464,17 @@ It returns the documents with full hits meta data including the real index in wh
                                         {:should (q/prepare-terms one-of)
                                          :minimum_should_match 1})
                           query (update :filter conj query_string)
-                          (seq date-range-query) (update :filter conj {:range date-range-query}))]
+                          (seq date-range-query) (update :filter conj {:range date-range-query}))
+            doc-query (-> es-params
+                          rename-sort-fields
+                          (with-default-sort-field props)
+                          make-es-read-params)]
```

Comment:
for the semantic of the variable name, the variable `doc-query` is the query parameters. 
```suggestion
            query-params (-> es-params
                             rename-sort-fields
                             (with-default-sort-field props)
                             make-es-read-params)]
```